tests/ci_build/Dockerfile.ubuntu1404_crosstool

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+MAINTAINER Aran Khanna <arankhan@amazon.com>
+
+# UPDATE BOX
```

Comment:
I am going to do a refactor after I push Android and iOS build tests, which rely on similar toolchains.